.github/workflows/release.yaml

```diff
@@ -26,7 +26,9 @@ jobs:
           echo "BRANCH=$(git rev-parse --abbrev-ref HEAD)" >> $GITHUB_ENV
           echo "FLUX_VERSION=$($(pwd)/tools/bin/stoml $(pwd)/tools/dependencies.toml flux.version)" >> $GITHUB_ENV
       - name: "Make All"
-        run: make all
+        run: |
+          make all
```

Comment:
How about splitting this into a separate step? That way its easier to tell which command fails.